### PR TITLE
Add new target TMTR-TMOTORF7_AIO config

### PR DESCRIPTION
**Pull request by tobeornottobe81**
Because of the angle of the gyroscope, 
we use 
set align_board_yaw = -45
configuration. What's the problem with it。
thank you.

### CODE REVIEW

File: configs/default/TMTR-TMOTORF7_AIO.config [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+
+
+# feature
+feature OSD
+
+# master
+set mag_bustype = I2C
```
Comment (haslinghuis) [inline configs/default/TMTR-TMOTORF7_AIO.config:111]: There is no MAG hardware #define

File: configs/default/TMTR-TMOTORF7_AIO.config [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+set mag_i2c_device = 2
+set baro_spi_device = 3
+set blackbox_device = SPIFLASH
+set current_meter = ADC
+set battery_meter = ADC
+set beeper_inversion = ON
+set beeper_od = OFF
+set osd_gps_sats_show_hdop = ON
```
Comment (haslinghuis) [inline configs/default/TMTR-TMOTORF7_AIO.config:119]: User setting

File: configs/default/TMTR-TMOTORF7_AIO.config [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+set max7456_spi_bus = 2
+set flash_spi_bus = 3
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set align_board_yaw = -45
+set pinio_box = 40,255,255,255
+
+
```
Comment (haslinghuis) [inline configs/default/TMTR-TMOTORF7_AIO.config:127]: Please remove one line

Comment (haslinghuis) [reply configs/default/TMTR-TMOTORF7_AIO.config:127]: Missing `set motor_pwm_protocol = DSHOT600`

Comment (tobeornottobe81) [reply configs/default/TMTR-TMOTORF7_AIO.config]: hi ，I have corrected the configuration file according to the use of hardware, please check them, thank you!

File: configs/default/TMTR-TMOTORF7_AIO.config (hunk 1 above)
Comment (tobeornottobe81) [reply configs/default/TMTR-TMOTORF7_AIO.config]: We don't have a magnetic compass on the board，We don't confirm whether the customer will use it，So we have to cancel it。

File: configs/default/TMTR-TMOTORF7_AIO.config (hunk 2 above)
Comment (tobeornottobe81) [reply configs/default/TMTR-TMOTORF7_AIO.config]: You mean cancel  set osd_gps_sats_show_hdop = ON ，OK。

**Review by haslinghuis — APPROVED** [review]

**Review by sugaarK — APPROVED** [review]

**Pull request merged**